src/state/mod.rs

```diff
@@ -68,18 +70,22 @@ impl From<ModProfile!["0.0.0"]> for ModProfile!["0.1.0"] {
 #[obake::versioned]
 #[obake(version("0.0.0"))]
 #[obake(version("0.1.0"))]
+#[obake(version("0.2.0"))]
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ModData {
     pub active_profile: String,
     #[obake(cfg("0.0.0"))]
     pub profiles: BTreeMap<String, ModProfile!["0.0.0"]>,
-    #[obake(cfg("0.1.0"))]
+    #[obake(cfg(">=0.1.0"))]
     pub profiles: BTreeMap<String, ModProfile!["0.1.0"]>,
-    #[obake(cfg("0.1.0"))]
+    #[obake(cfg(">=0.1.0"))]
     pub groups: BTreeMap<String, ModGroup>,
+    #[obake(cfg(">=0.2.0"))]
+    #[serde(with = "any_key_map")]
+    pub gameplay_affecting_cache: HashMap<ModInfo, GameplayAffecting>,
```

Comment:
Shouldn't this be in the `Cache` rather than user data?

Reply:
Also I think a more appropriate key would be `ModResolution` instead of `ModInfo`. Well specifically `ModResolution.url` which should be turned into a newtype if it's used for this. `ModResolution` is a fully resolved mod and doesn't carry all the other unnecessary mod metadata with it.

EDIT: I misremembered the details surrounding `ModResolution`. See my comment on it above.

Reply:
Moved the cache to `Cache`.